features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -423,5 +423,14 @@ data class RumConfiguration internal constructor(
             rumConfig = rumConfig.copy(rumSessionTypeOverride = rumSessionTypeOverride)
             return this
         }
+
+        /**
+         * Disables JankStats tracking. This flag may be enabled by cross-platform SDKs where
+         * native frame metrics are not meaningful.
+         */
+        internal fun setDisableJankStats(disable: Boolean): Builder {
```

Comment:
minor: it can be just `disableJankStats`, but let's keep it like that, it is only for us anyway.